src/workflows-app/SubmissionDetails.test.js

```diff
@@ -298,45 +196,45 @@ describe('Submission Details page', () => {
 
     // Assert
     const table = await screen.findByRole('table');
-    const rows = within(table).queryAllByRole('row');
+    const rows = within(table).getAllByRole('row');
     expect(rows.length).toBe(3);
 
-    const headers = within(rows[0]).queryAllByRole('columnheader');
+    const headers = within(rows[0]).getAllByRole('columnheader');
     expect(headers.length).toBe(3);
 
     // Act - click on sort button on Duration column to sort by ascending order
-    user.click(await within(headers[2]).findByRole('button'));
+    await user.click(await within(headers[2]).findByRole('button'));
 
     // Assert
     // check that rows are now sorted by duration in ascending order
-    const cellsFromDataRow1 = within(rows[1]).queryAllByRole('cell');
+    const cellsFromDataRow1 = within(rows[1]).getAllByRole('cell');
     expect(cellsFromDataRow1.length).toBe(3);
     await within(cellsFromDataRow1[0]).findByText('FOO1');
-    await within(cellsFromDataRow1[1]).findByText('Succeeded');
-    await within(cellsFromDataRow1[2]).findByText('37 seconds');
+    within(cellsFromDataRow1[1]).getByText('Succeeded');
+    within(cellsFromDataRow1[2]).getByText('37 seconds');
 
-    const cellsFromDataRow2 = within(rows[2]).queryAllByRole('cell');
+    const cellsFromDataRow2 = within(rows[2]).getAllByRole('cell');
     expect(cellsFromDataRow2.length).toBe(3);
     await within(cellsFromDataRow2[0]).findByText('FOO2');
```

Comment:
Same as previous comment. Do we need to `await`/`findBy` here?

Reply:
good catches - no, we don't. those were leftover from when I had misunderstood the way things were being rendered!